lib/providers/settings.dart

```diff
@@ -72,3 +72,5 @@ final settingsGraphicThemeProvider =
 );
 
 final randomSeedProvider = StateProvider<int>((ref) => 42);
+
+final settingsImageViewerAppProvider = StateProvider<String>((ref) => '');
```

Comment:
Can we please be consistent in naming - perhaps call this `imageViewerSettingsProvider`